src/__tests__/Search/__snapshots__/SearchResultsList.test.tsx.snap

```diff
@@ -169,7 +169,6 @@ exports[`SearchResultsList should match snapshot 1`] = `
             >
               <div
                 class="MuiListItemButton-root MuiListItemButton-gutters MuiButtonBase-root css-16ac5r2-MuiButtonBase-root-MuiListItemButton-root"
-                id="0"
```

Comment:
IDs were removed because they are no longer needed. previously the `useCheckRevision` hook was using the ID to add it to the checkedRevisions state, but now it stores the actual revision instead.